db.js

```diff
@@ -1,16 +1,130 @@
 #!/usr/bin/env node
 /* eslint no-unused-expressions: off */
 
-function create() {
+// set up the connection with db
+const pg = require('pg');
+const Sequelize = require('sequelize');
+
+const db = new Sequelize('db_name', 'username', 'password', { // change database name, username, password
+  host: 'host', // change host
+  dialect: 'postgres',
+});
+
+
+
+// definition of the Tweets table
+const Tweet = db.define('tweets', {
+  id: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  text: {
+    type: Sequelize.TEXT,
+  },
+  hashtag: {
+    type: Sequelize.ARRAY(Sequelize.STRING),
+  },
+  timestamp: {
+    type: Sequelize.DATE,
+  },
+  geo: {
+    type: Sequelize.STRING,
+  },
+  location: {
+    type: Sequelize.STRING,
+  },
+});
+
+// definition of the HashtagStats table
+const HashtagStats = db.define('hashtagstats', {
+  country: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  hashtag: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  count: {
+    type: Sequelize.INTEGER,
+    defaultValue: 1
+  },
+});
```

Comment:
I think each processor should have its own table that it knows about—I would put the stuff defining table structure for each metric into the code for the processor. E.g. this would go with the new `HashtagStats` processor you're going to create.